guidelines/terms/22/target-offset.html

```diff
@@ -1,6 +1,11 @@
 <dt class="new"><dfn data-lt="target offsets">target offset</dfn></dt>
 <dd class="new">
 	<p class="change">New</p>	
-   <p>the distance measured from the farthest point of a target (A), to the nearest point of the second target (B). The offset includes the target and spacing around the target.</p>
+   <p>size of a target (A) plus the spacing to a second target (B).</p>
    <p class="note">The offset from A to B may be different than the offset from B to A, if the sizes of these targets differ.</p>
+   <div class="note">
+      <p>For horizontally aligned targets, target offset is the width of target A plus the horizontal space to the closest edge of target B. For vertically aligned targets, target offset is the the height of target A plus the vertical space to the closest edge of target B. For targets that are neither horizontally nor vertically aligned, the target offset is the longest measure from any point of A to the nearest point of B.</p>
+
+      <p>Two targets are horizontally aligned if a horizontal line can be drawn that goes through both targets, but no vertical line can be drawn that goes through both targets. Two targets are vertically aligned if a vertical line can be drawn that goes through both targets, but no horizontal line can be drawn that goes through both targets.</p>
```

Comment:
This definition allows for some types of overlapping elements to be "aligned". I think this is okay, but only if the reader interprets the "closest edge of target B" as possibly being "the boundary of A and B where the overlap ends", ie, there is 0 space between them. If we go with this example, it would be good to add examples of overlapping elements which are aligned vs not to the understanding doc to clarify this.